ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/BerlinGasCalculator.java

```diff
@@ -203,4 +207,49 @@ public Gas calculateStorageRefundAmount(
       }
     }
   }
+
+  @Override
+  public Gas modExpGasCost(final Bytes input) {
+    // Typically gas calculations are delegated to a GasCalculator instance,
```

Comment:
This comment was in the old code, but I think we should remove it since we are doing exactly what it said we shouldn't. ;) - Because finally we have a good reason.